packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -388,7 +388,7 @@ const extractTemplate = (templateName, outputDir) => {
         cwd: p.join(tmp),
         sync: true
     })
-    sh.mv(p.join(tmp, templateName), outputDir)
+    sh.cp('-R', p.join(tmp, templateName), outputDir)
```

Comment:
Just as a summary of our discussion. I'd like to see some numbers showing whether or not this will be a significant slow down to project generation. Now that the delete command on line 392 will be deleting more than just an empty temp folder, I suspect that this kind of action can be relatively slow on windows machines specifically. 

If it is, maybe we discuss not deleting it at all?

Reply:
@bendvc Time generating a project `generator-shelljs-cp-to-extract-template` vs `release-1.4.x` branch.

Steps to measure the time per branch:
1.  Using Windows Command line checkout branch.
2. `git clean -dxf && npm ci`.
3. `set GENERATOR_PRESET=retail-react-app-demo`.
4. Repeat the process 3 times.

### Results:

#### `generator-shelljs-cp-to-extract-template` branch

```
C:\Users\User\git\pwa-kit>timecmd.bat node packages/pwa-kit-create-app/scripts/create-mobify-app.js --outputDir ./preset-retail-react-app-demo
Installing dependencies for the generated project...

To change your ecommerce back end you will need to update your storefront configuration. More information: https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/configuration-options

Successfully generated a project in ./preset-retail-react-app-demo
command took 0:0:59.11 (59.11s total)
```

```
C:\Users\User\git\pwa-kit>timecmd.bat node packages/pwa-kit-create-app/scripts/create-mobify-app.js --outputDir ./preset-retail-react-app-demo
Installing dependencies for the generated project...

To change your ecommerce back end you will need to update your storefront configuration. More information: https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/configuration-options

Successfully generated a project in ./preset-retail-react-app-demo
command took 0:0:52.47 (52.47s total)
```

```
C:\Users\User\git\pwa-kit>timecmd.bat node packages/pwa-kit-create-app/scripts/create-mobify-app.js --outputDir ./preset-retail-react-app-demo
Installing dependencies for the generated project...

To change your ecommerce back end you will need to update your storefront configuration. More information: https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/configuration-options

Successfully generated a project in ./preset-retail-react-app-demo
command took 0:0:53.28 (53.28s total)
```
----

#### `release-1.4.x` branch

```
C:\Users\User\git\v2-pwa-kit>timecmd.bat node packages/pwa-kit-create-app/scripts/create-mobify-app.js --outputDir ./preset-retail-react-app-demo
Installing dependencies for the generated project (this can take a while)

Successfully generated a project in ./preset-retail-react-app-demo
command took 0:0:59.40 (59.40s total)
```

```
C:\Users\User\git\v2-pwa-kit>timecmd.bat node packages/pwa-kit-create-app/scripts/create-mobify-app.js --outputDir ./preset-retail-react-app-demo
Installing dependencies for the generated project (this can take a while)

Successfully generated a project in ./preset-retail-react-app-demo
command took 0:0:54.47 (54.47s total)
```

```
C:\Users\User\git\v2-pwa-kit>timecmd.bat node packages/pwa-kit-create-app/scripts/create-mobify-app.js --outputDir ./preset-retail-react-app-demo
Installing dependencies for the generated project (this can take a while)

Successfully generated a project in ./preset-retail-react-app-demo
command took 0:0:57.39 (57.39s total)
```